reporter/src/funTest/assets/spdx-document-reporter-expected-output.spdx.json

```diff
@@ -10,5 +10,107 @@
   "name" : "some document name",
   "dataLicense" : "CC0-1.0",
   "comment" : "some document comment",
-  "documentNamespace" : "<REPLACE_DOCUMENT_NAMESPACE>"
+  "documentNamespace" : "<REPLACE_DOCUMENT_NAMESPACE>",
+  "packages" : [ {
+    "SPDXID" : "SPDXRef-Package-0-root-package",
+    "copyrightText" : "NOASSERTION",
+    "downloadLocation" : "NOASSERTION",
+    "filesAnalyzed" : false,
+    "homepage" : "NOASSERTION",
+    "licenseConcluded" : "NOASSERTION",
+    "licenseDeclared" : "NOASSERTION",
+    "name" : "Root package",
+    "summary" : "An artificial stub package describing the root of the dependency tree."
+  }, {
+    "SPDXID" : "SPDXRef-Package-1-first-package",
+    "copyrightText" : "Copyright 2020 Some copyright holder\nCopyright 2020 Some other copyright holder",
+    "downloadLocation" : "https://some-host/first-package.jar",
+    "filesAnalyzed" : false,
+    "homepage" : "first package's hompage URL",
+    "licenseConcluded" : "NOASSERTION",
+    "licenseDeclared" : "BSD-3-Clause AND (MIT OR GPL-2.0-only)",
+    "name" : "Maven:first-package-group:first-package",
+    "summary" : "A package with all supported attributes set, with a VCS URL containing auser name, and with a scan result containing two copyright finding matched to a license finding.",
+    "versionInfo" : "0.0.1"
+  }, {
+    "SPDXID" : "SPDXRef-Package-2-first-package-vcs",
+    "copyrightText" : "Copyright 2020 Some copyright holder\nCopyright 2020 Some other copyright holder",
+    "downloadLocation" : "git+ssh://github.com/path/first-package-repo.git@deadbeef#project-path",
+    "filesAnalyzed" : false,
+    "homepage" : "first package's hompage URL",
+    "licenseConcluded" : "NOASSERTION",
+    "licenseDeclared" : "NOASSERTION",
+    "name" : "Maven:first-package-group:first-package",
+    "summary" : "A package with all supported attributes set, with a VCS URL containing auser name, and with a scan result containing two copyright finding matched to a license finding.",
+    "versionInfo" : "0.0.1"
+  }, {
+    "SPDXID" : "SPDXRef-Package-3-first-package-source-artifact",
+    "copyrightText" : "Copyright 2020 Some copyright holder\nCopyright 2020 Some other copyright holder",
+    "downloadLocation" : "https://some-host/first-package-sources.jar",
+    "filesAnalyzed" : true,
+    "homepage" : "first package's hompage URL",
+    "licenseConcluded" : "NOASSERTION",
+    "licenseDeclared" : "NOASSERTION",
+    "name" : "Maven:first-package-group:first-package",
+    "summary" : "A package with all supported attributes set, with a VCS URL containing auser name, and with a scan result containing two copyright finding matched to a license finding.",
+    "versionInfo" : "0.0.1"
+  }, {
+    "SPDXID" : "SPDXRef-Package-4-forth-package",
```

Comment:
forth => fourth. Can you reorder the entries so they are alphabetically sorted?

Reply:
The order of the entries is aligned with the spec and I don't want to re-order them.

Reply:
Typo fixed.